homeassistant/components/homematicip_cloud/strings.json

```diff
@@ -5,7 +5,7 @@
         "title": "Pick HomematicIP Access point",
         "data": {
           "hapid": "Access point ID (SGTIN)",
-          "pin": "Pin Code (optional)",
+          "pin": "[%key:common::config_flow::data::pin%]",
```

Comment:
Optional is gone ?

Reply:
Yeah I am not sure about that part either.
@SukramJ can we remove that part or is necessary for users to understand the config flow?

Reply:
that would lead to partial translation with reference strings again 😛 But in this case that's easier I guess.

Reply:
Can't we add "optional" as common string? And format a new string using two common strings (pin and optional)

Reply:
I think it is acceptable to remove "optional".

Reply:
Thanks for your input @SukramJ

Reply:
The UI should indicate it's optional so it shouldn't be necessary to be included in the string